packages/user-interface/package.json

```diff
@@ -1,94 +1,94 @@
-  },
-  "peerDependencies": {
-    "react": "^17.0.2"
-  },
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "^20.0.0",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.12.38",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "babel-eslint": "^10.0.3",
-    "cross-env": "^7.0.2",
-    "gh-pages": "^2.2.0",
-    "microbundle-crl": "^0.13.10",
-    "npm-run-all": "^4.1.5",
-    "react": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "rimraf": "^3.0.2",
-    "sass": "^1.38.2",
-    "typescript": "^4.1.2",
-    "wait-on": "^6.0.1"
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js"
+    },
+    "./style.css": "./dist/style.css"
   },
   "files": [
-    "dist"
+    "dist",
+    "package.json"
   ],
+  "scripts": {
+    "build": "pnpm wait && tsc && vite build",
+    "clean": "rimraf dist",
+    "dev": "pnpm wait && vite build --watch --mode development",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "preinstall": "npx only-allow pnpm",
+    "wait": "wait-on ../api/dist/index.js && wait-on ../authentication/dist/index.js && wait-on ../localization/dist/index.js",
+    "test": "vitest run",
+    "test:watch": "vitest --watch"
+  },
   "dependencies": {
     "@apollo/client": "^3.4.11",
     "@formio/react": "^5.2.3",
-    "@gemeente-denhaag/action": "0.1.1-alpha.11",
-    "@gemeente-denhaag/alert": "0.1.1-alpha.278",
-    "@gemeente-denhaag/card": "0.2.3-alpha.348",
-    "@gemeente-denhaag/components-react": "0.1.1-alpha.305",
-    "@gemeente-denhaag/contact-timeline": "0.1.1-alpha.25",
-    "@gemeente-denhaag/descriptionlist": "0.1.1-alpha.234",
-    "@gemeente-denhaag/icons": "0.2.3-alpha.308",
-    "@gemeente-denhaag/link": "0.2.3-alpha.70",
-    "@gemeente-denhaag/page": "0.1.1-alpha.223",
-    "@gemeente-denhaag/process-steps": "0.1.0-alpha.196",
-    "@gemeente-denhaag/responsive-content": "0.1.1-alpha.168",
-    "@gemeente-denhaag/sidenav": "0.1.0-alpha.126",
-    "@gemeente-denhaag/tab": "0.2.3-alpha.348",
-    "@gemeente-denhaag/table": "0.1.1-alpha.228",
-    "@gemeente-denhaag/textfield": "0.2.3-alpha.70",
-    "@nl-portal/nl-portal-api": "^0.1.1-alpha.41",
-    "@nl-portal/nl-portal-authentication": "^0.1.1-alpha.41",
-    "@nl-portal/nl-portal-localization": "^0.1.1-alpha.41",
+    "@gemeente-denhaag/action": "0.1.1-alpha.28",
+    "@gemeente-denhaag/alert": "0.1.1-alpha.332",
+    "@gemeente-denhaag/card": "0.2.3-alpha.362",
+    "@gemeente-denhaag/components-react": "0.1.1-alpha.300",
+    "@gemeente-denhaag/contact-timeline": "0.1.1-alpha.20",
+    "@gemeente-denhaag/descriptionlist": "0.1.1-alpha.262",
+    "@gemeente-denhaag/icons": "0.2.3-alpha.362",
+    "@gemeente-denhaag/link": "0.2.3-alpha.362",
```

Comment:
In deze versie van de link packages zit nog een bug dacht ik, graag even checken of links nog goed werken en anders even de oude versie laten staan.